src/morale.h

```diff
@@ -4,8 +4,16 @@
 #include "json.h"
 #include <string>
 #include "calendar.h"
+#include "effect.h"
+#include "bodypart.h"
+
+#include <functional>
+
+class item;
+class player;
 
 struct itype;
+struct morale_mult;
 
 enum morale_type : int {
```

Comment:
It looks like morale.h is included in a lot of places, probably only to provide this enum (I noticed because I( changed something in morale.h and rebuilt, and it's rebuilding a significant fraction of the game).  Probably worthwhile to split it out into its own header file.